Osiris/Config.cpp

```diff
@@ -226,6 +226,12 @@ void Config::load(size_t id) noexcept
             espConfig.distanceColor[1] = espJson["Distance color"][1].asFloat();
             espConfig.distanceColor[2] = espJson["Distance color"][2].asFloat();
         }
+		if (espJson.isMember("activeWeapon")) espConfig.distance = espJson["activeWeapon"].asBool();
```

Comment:
JSON key names should start with capital letter see above.

Reply:
okay I will keep this in mind. I thaught Capital in Json means that the Variable is explicit Exposed. Isn't it?